.github/stale.yml

```diff
@@ -0,0 +1,17 @@
+# Number of days of inactivity before an issue becomes stale
+daysUntilStale: 60
+# Number of days of inactivity before a stale issue is closed
+daysUntilClose: 7
+# Issues with these labels will never be considered stale
+exemptLabels:
+  - pinned
+  - security
+# Label to use when marking an issue as stale
+staleLabel: wontfix
+# Comment to post when marking an issue as stale. Set to `false` to disable
+markComment: >
+  This issue has been automatically marked as stale because it has not had
+  recent activity. It will be closed if no further activity occurs. Thank you
+  for your contributions.
+# Comment to post when closing a stale issue. Set to `false` to disable
+closeComment: false
```

Comment:
Not sure I understand the `closeComment`.
Does it mean that when I set it to `true` a comment will be posted when a stale issue is closed?
But what comment?

Reply:
"This issue was exterminated!"

Reply:
I think it's as you write, but I don't understand that as well. There is no documentation with the explanation -- I only copied their example.

We can try the default one and tweak it if needed.